requirements/dev.txt

```diff
@@ -26,7 +26,7 @@ jinja2-pluralize==0.3.0   # via diff-cover
 jinja2==2.10              # via diff-cover, jinja2-pluralize
 markupsafe==1.0           # via jinja2
 packaging==16.8           # via caniusepython3
-path.py==10.5             # via edx-i18n-tools
+path.py>=8.2.1            # via edx-i18n-tools
```

Comment:
Why do we want to update the version on this one?

Reply:
@iloveagent57

Reply:
@schenedx We are matching completion's `path.py` requirements with edx-platform's.